docs/4.3/enterprise/quickstart-enterprise.md

```diff
@@ -315,6 +320,101 @@ Any SAML-compliant provider can be configured with Teleport by following the
 same steps.  There are Teleport Enterprise customers who are using Oracle IDM,
 SailPoint and others.
 
+## Run Teleport Enterprise using Docker
```

Comment:
Add a note (somewhere in this section) that this won't let you SSH onto the host. It's probably not intended for prod usage on nodes.